app/controllers/api/v1/users_controller.rb

```diff
@@ -14,9 +14,9 @@ def show
   def index
     if params[:user][:search]
       role = if current_user.shop?
-        2
+        "Shipper"
       elsif current_user.shipper?
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/ElseAlignment
**Severity**: INFO
**File**: app/controllers/api/v1/users_controller.rb L18

Align `elsif` with `if`.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*